TicTacToe.js

```diff
@@ -0,0 +1,320 @@
+//Readline bloiler plate code
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    readlineInterface.question(questionText, resolve);
+  });
+}
+
+let playerStatus = 1;
+rowOne = [' 1','|','2','|','3'];
+rowDashes = ['---','---','---'];
+rowTwo = [' 4','|','5','|','6'];
+rowThree = [' 7','|','8','|','9'];
+
+
+printBoard();
+console.log('\n' +  "Welcome to tic tac toe, a two player game.  The first player to guess will be X's and the second will be O's.  Please choose a game board section based on it corresponding number."+'\n')
+playerOne();
+
+//printBoard();
+
+function printBoard(){
+  let rowOneString = rowOne.join(' ');
+  console.log(rowOneString);
+  let rowOfDashes = rowDashes.join(' ');
+  console.log(rowOfDashes);
+  let rowTwoString = rowTwo.join(' ');
+  console.log(rowTwoString);
+  console.log(rowOfDashes);
+  let rowThreeString = rowThree.join(' ');
+  console.log(rowThreeString);
+}
+
+function testMove(testNumber,playerNumber){
```

Comment:
another minor quibble: the purpose of the `testNumber` param was not clear to me on first read; might be better named `cellNumber` or `cellIndex` -- since (a) the number itself is not a test, and (b) the fact that it's a number is true, but less salient than the fact that it's an index.